### PR TITLE
#21 assign item to a category

### PR DESCRIPTION
**Pull request by MariaKohnen**
added new dropdown on itemEdit component to choose a category. The default value is "no category".
The items overview get sorted by category.
backend: reduce to one item dto because they get reduntant

### CODE REVIEW

File: frontend/src/components/packingItemDetails/CategoryDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    {categories.map(item => (
+                            <li className="dd-list-item" key={item}>
+                                <button type="button" onClick={() => handleOnClick(item)}>
+                                    {item}
+                                </button>
+                            </li>))}
+                </ul>)}
+        </div>
+    )
+}
```
Comment (Boltzmann) [inline frontend/src/components/packingItemDetails/CategoryDropdown.tsx:50]: You wrote this and the Status Dropdown yourself? Interesting...

Comment (Boltzmann) [reply frontend/src/components/packingItemDetails/CategoryDropdown.tsx:50]: OK, after a short search I get the feeling, that this is what one has to do to get a dropdown menu.

**Review by Boltzmann — APPROVED** [review]
Not much to say.

File: frontend/src/components/packingItemDetails/CategoryDropdown.tsx (hunk 1 above)
Comment (MariaKohnen) [reply frontend/src/components/packingItemDetails/CategoryDropdown.tsx:50]: I think there is also an html element, but I definitely didn't want a dropdown from a library. I have therefore rebuilt and adapted it based on a tutorial and some similar features in my app.

**Pull request merged**